usuarios/delete-usuarios.spec.js

```diff
@@ -0,0 +1,31 @@
+const user = require('../utils/userData')
+const testServer = require('../utils/testServer')
+const rota = require('../utils/rotas')
+
+var usuario
+var user_id
+
+beforeEach( async (done) => {
+  usuario = user.dadosDoUsuario()
+
+  const response_user = await user.criarUsuario(usuario)
+
+  user_id = await response_user.body._id
+  done()
```

Comment:
não precisa desse done, o async/await já se resolve sozinho

```suggestion
beforeEach( async () => {
  usuario = user.dadosDoUsuario()

  const response_user = await user.criarUsuario(usuario)

  user_id = await response_user.body._id
```